src/Models/Constants/Constants.ts

```diff
@@ -542,3 +543,20 @@ export const RequiredAccessRoleGroupForADTInstance: AzureAccessPermissionRoleGro
 };
 
 export const CONNECTION_STRING_SUFFIX = '.kusto.windows.net';
+
+/** Quick time span key to value in millisecond mapping */
+export const QuickTimeSpans = {
```

Comment:
nit: can we name this more generically since it's really just a lookup for time durations? Or does this object have to only contain these entries?

Reply:
If it is specific to that control, can we move it to a contants file or that component instead of the global contants? (same for the enum)

Reply:
yeah it is a specific control, when I was moving the things around I forgot to move those to that standalone component, will move it there now.

Reply:
oh in fact, I just noticed that why I kept those here as globals was because of the getQuickTimeSpanKeyByValue method which is used by multiple components but it also uses QuickTimeSpans inside, that is why QuickTimeSpans was in Constants. But I will move it to that component, let me know if it is okay (I know that we talked about not having methods in Utils which do imports from specific components type files but in this case I think it make sense). 2795f0678dc26486dfb58a6995891ae8184f9924

Reply:
can we move the util to that component as well? 🤔